consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java

```diff
@@ -164,17 +171,39 @@ public SubmitQueryResponse submitQuery(ISubmitQueryRequest submitQueryRequest,
         String xmlContent = SdtContext.getContext().getRawOutXml();
         String fromDate = xmlElementValueReader.getElementValue(xmlContent, FROM_DATE);
         String toDate = xmlElementValueReader.getElementValue(xmlContent, TO_DATE);
-        ClaimDefencesResponse response = claimDefences.claimDefences(idamRepository.getSdtSystemUserAccessToken(),
-                                                                     s2SRepository.getS2SToken(),
-                                                                     SdtContext.getContext().getCustomerIdamId(),
-                                                                     fromDate,
-                                                                     toDate);
-
         SubmitQueryResponse submitQueryResponse = new SubmitQueryResponse();
-        submitQueryResponse.setClaimDefencesResults(response.getResults());
-        submitQueryResponse.setClaimDefencesResultsCount(response.getResultCount());
+        try {
+            ClaimDefencesResponse response = claimDefences.claimDefences(idamRepository.getSdtSystemUserAccessToken(),
+                                                                         s2SRepository.getS2SToken(),
+                                                                         SdtContext.getContext().getCustomerIdamId(),
+                                                                         fromDate,
+                                                                         toDate);
+            updateResponseStatus(submitQueryRequest, submitQueryResponse, response);
+        } catch (FeignException e) {
+            boolean noResults = NO_DATA == e.status() || NOT_FOUND == e.status();
+            if (!noResults) {
```

Comment:
Could not noResults be replaced with a positively named variable instead?  Something like resultsExist.  It would make the logic a bit easier to understand.

Reply:
NO_DATA == e.status() || NOT_FOUND == e.status();
 these status codes mean, no data

Reply:
It was more the use of a double negative.  Double negatives are a bit harder to understand, so I wondered if there was a way of changing it to a positive.  For example, "if (resultsExist)" is easier to understand than "if (not noResults)" despite them meaning the same thing.